x-pack/legacy/plugins/maps/public/layers/styles/components/vector/legend/symbol_icon.js

```diff
@@ -15,31 +15,35 @@ export class SymbolIcon extends Component {
     imgDataUrl: undefined,
     prevSymbolId: undefined,
     prevFill: undefined,
+    prevStroke: undefined,
+    prevStrokeWidth: undefined,
   }
 
   componentDidMount() {
     this._isMounted = true;
-    this._loadSymbol(this.props.symbolId, this.props.fill);
+    this._loadSymbol(this.props.symbolId, this.props.fill, this.props.stroke, this.props.strokeWidth);
```

Comment:
Update `SymbolIcon.propTypes` definition at end of file to include `stroke` and `strokeWidth`